tests/e2e/utils/KubernetesCommandLineToolsExecutor.ts

```diff
@@ -89,6 +94,27 @@ export class KubernetesCommandLineToolsExecutor extends ShellExecutor {
         this.execWithLog(`${this.KUBERNETES_COMMAND_LINE_TOOL} create namespace ${this.namespace}`);
     }
 
+    createProject(projectName: string, timeout: number = TimeoutConstants.TS_SELENIUM_TERMINAL_DEFAULT_TIMEOUT * 2): void {
+        Logger.debug(`${this.getLoggingName(this.createProject.name)}: Create new project "${projectName}".`);
+        const polling: number = TestConstants.TS_SELENIUM_DEFAULT_POLLING;
+        const attempts: number = Math.ceil(timeout / polling);
+        let response: ShellString;
+        for (let i: number = 0; i < attempts; i++) {
```

Comment:
What do you think about replacement of lines 99-110 including whole `for-loop` with one command:
```
exec(`${this.KUBERNETES_COMMAND_LINE_TOOL} new-project ${projectName} -n ${this.namespace} || true`)
```

Reply:
Answer above